src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx

```diff
@@ -50,6 +49,9 @@ export default class FinalSubmissionQuestions extends React.Component<Props, Con
   updateSigned = () => this.setState({ signed: !this.state.signed })
   updateCert = () => this.setState({ certificateOfAuth: !this.state.certificateOfAuth })
 
+  updateEditionSize = text => this.setState({ editionInfo: { ...this.state.editionInfo, size: text } })
```

Comment:
Maybe to be explicit (since convection expects an integer for this field) we can rename the argument to `size` or `value`? (#minor). Only because as I was reading this, `text` suggests it would be a `string` input 😄

Reply:
Ah, looks like this is wrong in metaphysics then, as it expects a string

Reply:
Oh shucks I thought we caught that 😞

Reply:
Shipped a PR to metaphysics for it: https://github.com/artsy/metaphysics/pull/767